tasks/run-file-updates/run-file-updates.yaml

```diff
@@ -76,26 +88,30 @@ spec:
                            -p ref="${ref}" \
                            -p paths="${updatedPaths}" \
                            -p application=${application} \
+                           -p file_updates_secret="${file_updates_secret}" \
                            -s "$(params.synchronously)" \
```

Comment:
This param doesn't even exist? I'm surprised it doesn't fail on this. This should probably be hardcoded anyway, no?

Reply:
maybe in a future update.